frontend/src/bundles/common/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,64 @@
+import { useAppSelector } from '~/bundles/common/hooks/hooks.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+type Properties = {
+    size: ValueOf<typeof AvatarSize>;
+};
+
+const AvatarSize = {
+    SMALL: 'sm',
+    MEDIUM: 'md',
+    LARGE: 'lg',
+} as const;
+
+const sizeToClass: Record<ValueOf<typeof AvatarSize>, string> = {
+    [AvatarSize.SMALL]: 'h-12 w-12',
+    [AvatarSize.MEDIUM]: 'h-20 w-20',
+    [AvatarSize.LARGE]: 'h-30 w-30',
+};
+
+const getTextSize = (size: ValueOf<typeof AvatarSize>): string => {
+    switch (size) {
+        case AvatarSize.SMALL: {
+            return 'text-xl';
+        }
+        case AvatarSize.MEDIUM: {
+            return 'text-3xl';
+        }
+        case AvatarSize.LARGE: {
+            return 'text-4xl';
+        }
+        default: {
+            return 'text-base';
+        }
+    }
+};
```

Comment:
```suggestion
const getTextSize = (size: ValueOf<typeof AvatarSize>): string => {
   const sizeMap: Record<ValueOf<typeof AvatarSize>, string> = {
        [AvatarSize.SMALL]: 'text-xl',
        [AvatarSize.MEDIUM]: 'text-3xl',
        [AvatarSize.LARGE]: 'text-4xl',
    };

    return sizeMap[size] || 'text-base';
};
```
I think it's better to rewrite it like this. And move to helper folder.

Reply:
I think even the helper function is unnecessary, sizeMap[size] could be used directly without the function call.

Reply:
Thanks) Changed as you said

Reply:
Merge development into your branch.